src/nordictrackelliptical.cpp

```diff
@@ -1197,74 +1248,74 @@ void nordictrackelliptical::btinit() {
             QThread::msleep(400);
             writeCharacteristic(noOpData10c, sizeof(noOpData10c), QStringLiteral("init"), false, false);
             QThread::msleep(400);
-        } else if (!proform_hybrid_trainer_xt) {
-            writeCharacteristic(initData10, sizeof(initData10), QStringLiteral("init"), false, false);
+        } else if (proform_hybrid_trainer_xt) {
```

Comment:
again why you changed this condition? you should change only the nordictrack_elliptical_c7_5 case

Reply:
Same reason as before.